### DISCUSSION TITLE
Add the APK download on the website

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [SuperTux/supertux#2615] [Ideas]
Hi. Google Play isn't working on my mobile phone, so I have a suggestion to add an APK download on the website.

**Comment by ghost** [comment]
I agree. I would like to have this too. An older android tablet I have is also not having a working google play and as a consequence I can't install supertux on it.
A link to the APK would be very useful!

**Reply by mrkubax10** [reply]
Yeah this could be done, for example on website in Download > Android under Google Play link.
